### PR TITLE
Draw arc with GL triangle strip instead of texture based

### LINKED ISSUE #251: Arc renderer uses texture
https://github.com/Wynntils/Artemis/blob/6cc9ce1319b0083464b4a7e21ad3f8f25d876a52/common/src/main/java/com/wynntils/mc/render/RenderUtils.java#L352

Using a texture to draw an arc is undesirable.

### PR DESCRIPTION
**Pull request by HighCrit**
Fixes #251

### CODE REVIEW

File: common/src/main/java/com/wynntils/mc/render/RenderUtils.java [hunk 1]
```diff
@@ -337,33 +338,46 @@ public static void drawTexturedRectWithColor(
...
-        drawArc(new PoseStack(), color, x, y, z, fill, radius);
+    public static void drawArc(CustomColor color, int x, int y, int z, float fill, int innerRadius, int outerRadius) {
+        drawArc(new PoseStack(), color, x, y, z, fill, innerRadius, outerRadius);
     }
 
-    public static void drawArc(PoseStack poseStack, CustomColor color, int x, int y, int z, float fill, int radius) {
+    public static void drawArc(
+            PoseStack poseStack, CustomColor color, int x, int y, int z, float fill, int innerRadius, int outerRadius) {
         // keeps arc from overlapping itself
         int segments = (int) Math.min(fill * MAX_CIRCLE_STEPS, MAX_CIRCLE_STEPS - 1);
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/mc/render/RenderUtils.java:348]: Can't we increase MAX_CIRCLE_STEPS now?

Comment (HighCrit) [reply common/src/main/java/com/wynntils/mc/render/RenderUtils.java:348]: It's the same step count as Wynntils so 🤷

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/mc/render/RenderUtils.java:348]: Ah, it should be fine then.

**Review by kristofbolyai — APPROVED** [review]

**Comment by magicus** [comment]
@kristofbolyai Shouldn't we remove the arc texture as well?

**Comment by kristofbolyai** [comment]
Forgot to check that. Oops

**Comment by magicus** [comment]
Ah, sorry, it was @HighCrit's PR. I meant to tag the author, not blame the reviewer. :) It's just you've been so prolific lately that I kind of just assume all new PRs are from you. :)

**Comment by HighCrit** [comment]
> Ah, sorry, it was @HighCrit's PR. I meant to tag the author, not blame the reviewer. :) It's just you've been so prolific lately that I kind of just assume all new PRs are from you. :)

Ah yes, sorry. Totally missed that

**Pull request merged**